iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/digitaltwin/simulator/TestDigitalTwinDevice.java

```diff
@@ -0,0 +1,63 @@
+package tests.integration.com.microsoft.azure.sdk.iot.digitaltwin.simulator;
+
+import com.microsoft.azure.sdk.iot.device.ClientOptions;
+import com.microsoft.azure.sdk.iot.device.DeviceClient;
+import com.microsoft.azure.sdk.iot.device.IotHubClientProtocol;
+import com.microsoft.azure.sdk.iot.service.Device;
+import com.microsoft.azure.sdk.iot.service.RegistryManager;
+import com.microsoft.azure.sdk.iot.service.auth.AuthenticationType;
+import com.microsoft.azure.sdk.iot.service.exceptions.IotHubException;
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.extern.slf4j.Slf4j;
+import tests.integration.com.microsoft.azure.sdk.iot.digitaltwin.helpers.E2ETestConstants;
+import tests.integration.com.microsoft.azure.sdk.iot.helpers.Tools;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+
+@Slf4j
+@Getter
+public class TestDigitalTwinDevice {
```

Comment:
Would suggest renaming to DigitalTwinTestDevice

Reply:
We might also reuse this - TwinPnPTestInstance that already exists.

Reply:
refactored some code and do not need the other class anymore